roles/realm_join/tasks/main.yml

```diff
@@ -0,0 +1 @@
+- include_tasks: realm_join_ad.yml
```

Comment:
what's the reason to have all tasks in another file? prep-work for `realm_join_ipa.yml`?

Reply:
yeah, doesn't make much sense xD I think first I was thinking it could run the installer afterwards, but then decided against it.
So now it's tree steps, get the pexpect on the necessary version, join the realm and then prepare the keytabs for apache.